remoulade/api/main.py

```diff
@@ -1,14 +1,34 @@
 """ This file describe the API to get the state of messages """
 from flask import Flask, request
+from marshmallow import Schema, ValidationError, fields, validate, validates
 from werkzeug.exceptions import BadRequest, HTTPException, NotFound
 
 import remoulade
-from remoulade import get_scheduler
+from remoulade import get_broker, get_scheduler
 from remoulade.errors import RemouladeError
 from remoulade.state import StateNamesEnum
 
 app = Flask(__name__)
 
+"""
+    Class used to validate the data passed to
+    enqueue a message.
+"""
+
+
+class MessageSchema(Schema):
```

Comment:
this file is starting to be long put this in another file. schema next to this one